### PR TITLE
Task 15

### PR DESCRIPTION
**Pull request by RuslanGabd**

### CODE REVIEW

File: HelloWorld.class
Comment (Mark005) [inline HelloWorld.class:1]: why do you need this?

File: HelloWorld.java
Comment (Mark005) [inline HelloWorld.java:1]: and this

File: Screenshot.jpg
Comment (Mark005) [inline Screenshot.jpg:1]: ? application should response on http method get, so you shout take the response from requests to your service by the specific URI like here
image

**Review by Mark005 — CHANGES_REQUESTED** [review]
please check if your application works as expected and make a screenshot in PR comments

File: HelloWorld.class
Comment (RuslanGabd) [reply HelloWorld.class:1]: I made a mistake in understanding the line from the task “The branch must be from the first commit of the main branch.” and made a branch from Task_1.
I deleted both files

File: Screenshot.jpg
Comment (RuslanGabd) [reply Screenshot.jpg:1]: image

**Review by Mark005 — APPROVED** [review]
good job!

**Pull request closed without merging**